packages/design-system-tailwind-preset/package.json

```diff
@@ -51,11 +51,15 @@
     "@types/jest": "^27.4.1",
     "deepmerge": "^4.2.2",
     "jest": "^27.5.1",
+    "tailwindcss": "^3.4.13",
     "ts-jest": "^27.1.4",
     "typedoc": "^0.24.8",
     "typedoc-plugin-missing-exports": "^2.0.0",
     "typescript": "~5.2.2"
   },
+  "peerDependencies": {
+    "tailwindcss": "^3.4.13"
+  },
   "engines": {
     "node": "^18.18 || >=20"
```

Comment:
We should consider removing the Node-specific version constraints to avoid introducing unnecessary dependency requirements into other projects.